### PR TITLE
model and migration files

### PR DESCRIPTION
**Pull request by darrenclee**
I didn't merge into my development branch since this is one of our first pull request, we can start doing this tomorrow when we start merging

### CODE REVIEW

File: db/migrate/20161124041953_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :users do |t|
+      t.string :username
+      t.string :email
+      t.string :password_digest
```
Comment (whyderrick) [inline db/migrate/20161124041953_create_users.rb:6]: this is the only instance of password_digest in this code and I've never seen the term before. Can you explain it?

**Review by julesmartin76000 — COMMENTED** [review]
okay good

File: db/migrate/20161124043230_create_walks.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      t.integer :requester_id
+      t.integer :guardian_id
+
+      t.timestamps
+    end
+
+    add_foreign_key :walks, :users, column: :requester_id
+    add_foreign_key :walks, :users, column: :guardian_id
+  end
+end
```
Comment (julesmartin76000) [inline db/migrate/20161124043230_create_walks.rb:17]: good

**Review by julesmartin76000 — COMMENTED** [review]
good to go

**Review by xdkernelx — APPROVED** [review]
approved

**Review by whyderrick — APPROVED** [review]
done

**Pull request merged**